src/txdb.h

```diff
@@ -84,8 +86,6 @@ class CCoinsViewDBCursor: public CCoinsViewCursor
     void Next() override;
 
 private:
-    CCoinsViewDBCursor(CDBIterator* pcursorIn, const uint256 &hashBlockIn):
-        CCoinsViewCursor(hashBlockIn), pcursor(pcursorIn) {}
     std::unique_ptr<CDBIterator> pcursor;
     std::pair<char, COutPoint> keyTmp;
 
```

Comment:
line 92 still needed?

Reply:
Yes, for e.g. `pcursor` access in `CCoinsViewDB::Cursor()`.